# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openglad/fix-return-value.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Well - people should know how to return the status in main - especially
when they use it in the Makefile themselfes ...

  - Rene Rebe <rene@exactcode.de>

--- openglad-0.98/util/gladpack.c.orig	2004-03-14 19:07:59.000000000 +0100
+++ openglad-0.98/util/gladpack.c	2004-03-14 19:07:17.000000000 +0100
@@ -16,6 +16,7 @@
 		printf("Usage: gladpack (p|u) ...\n");
 		return 1;
 	}
+	return 0;
 }
 
 int pack(int argc, char **argv)
